dwave/cloud/config.py

```diff
@@ -193,6 +309,42 @@ def load_profile_from_files(filenames=None, profile=None):
 
         :exc:`ValueError`:
             Profile name not found.
+
+    Examples:
+        This example loads a profile based on configurations from two files. It
+        finds the first profile, dw2000A, in the first file, dwave_a.conf, and adds to
+        the values of the defaults section, overwriting the existing client value,
+        while ignoring the profile in the second file, dwave_b.conf.
+
+        The files, which are located in the current working directory, are
+        (1) dwave_a.conf::
+
+            [defaults]
+            endpoint = https://url.of.some.dwavesystem.com/sapi
+            client = qpu
+            token = ABC-123456789123456789123456789
+
+            [dw2000A]
+            client = sw
+            solver = EXAMPLE_2000Q_SYSTEM_A
+            token = DEF-987654321987654321987654321
+
+        and (2) dwave_b.conf::
+
+            [dw2000B]
+            endpoint = https://url.of.some.other.dwavesystem.com/sapi
+            client = qpu
+            solver = EXAMPLE_2000Q_SYSTEM_B
+
+        The following example code loads profile values from parsing both these files.
+
+        >>> import dwave.cloud as dc
+        >>> dc.config.load_profile_from_files(["./dwave_a.conf", "./dwave_b.conf"])   # doctest: +SKIP
+        {'client': u'sw',
+         'endpoint': u'https://url.of.some.dwavesystem.com/sapi',
+         'solver': u'EXAMPLE_2000Q_SYSTEM_A',
+         'token': u'DEF-987654321987654321987654321'}
+
```

Comment:
To clarify the behaviour of profile loading function, I would add:

    >>> dc.config.load_profile_from_files(["./dwave_a.conf", "./dwave_b.conf"], profile='dw2000B')
    ...

Also, in the current example, it would be useful to note that although `dwave_b.conf` overrides `dwave_a.conf`, the function by default returns *the first* profile (here `dw2000A`), if not specified otherwise.